### PR TITLE
added dto & mappers & controllers for Movie, MovieSession, User Authe…

### PR DESCRIPTION
**Pull request by AnaLanda**
…ntication

### CODE REVIEW

File: src/main/java/com/cinema/controllers/MovieController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public MovieController(MovieService movieService, MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
+
+    @PostMapping
+    public Movie addMovie(@RequestBody MovieRequestDto movieRequestDto) {
+        log.info("Trying to add MovieRequestDto " + movieRequestDto);
+        Movie movie = movieService.add(movieMapper.mapToMovie(movieRequestDto));
+        log.info("Successfully added MovieRequestDto " + movieRequestDto);
```
Comment (andriylagun) [inline src/main/java/com/cinema/controllers/MovieController.java]: If you have logger in Dao layer(in add method), do you need that here? And you adding Movie, not MovieRequestDto

File: src/main/java/com/cinema/controllers/MovieSessionController.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public MovieSession addMovieSession(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        log.info("Trying to add MovieSessionRequestDto " + movieSessionRequestDto);
+        MovieSession movieSession = movieSessionService.add(
+                movieSessionMapper.mapToMovieSession(movieSessionRequestDto));
+        log.info("Successfully added MovieRequestDto " + movieSessionRequestDto);
```
Comment (andriylagun) [inline src/main/java/com/cinema/controllers/MovieSessionController.java]: same for other controllers

**Review by andriylagun — APPROVED** [review]
Minor comments, but still good job!!

File: src/main/java/com/cinema/controllers/MovieController.java (hunk 1 above)
Comment (AnaLanda) [reply src/main/java/com/cinema/controllers/MovieController.java]: You're right!

File: src/main/java/com/cinema/service/mapper/UserMapper.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.cinema.service.mapper;
+
+import com.cinema.model.User;
+import com.cinema.model.dto.UserRequestDto;
+
+public class UserMapper {
+    public User mapToUser(UserRequestDto userRequestDto) {
```
Comment (Sofasmile) [inline src/main/java/com/cinema/service/mapper/UserMapper.java]: do we use this method? let's remove UserMapper class

File: src/main/java/com/cinema/model/dto/MovieResponseDto.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class MovieResponseDto {
+    private String title;
```
Comment (Sofasmile) [inline src/main/java/com/cinema/model/dto/MovieResponseDto.java:8]: let's add id field to all response dto

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**